apps/site/src/content/blog/astrostudio-cms.mdx

```diff
@@ -0,0 +1,50 @@
+# Introducing AstroStudio CMS: A Community-Driven CMS for Astro Studio
+
+As part of our efforts, we're excited to introduce AstroStudio CMS - a dedicated content management system (CMS) built on top of Astro's latest product, [Astro Studio](https://studio.astro.build). This project was developed by [Adam](https://github.com/Adammatthiesen), [Dreyfus](https://github.com/dreyfus92), and [Jumper](https://github.com/jdtjenkins), three passionate members of the Astro community.
```

Comment:
```suggestion
We're excited to introduce AstroStudio CMS - a dedicated content management system (CMS) built on top of [Astro Studio](https://studio.astro.build). This project was developed by [Adam](https://github.com/Adammatthiesen), [Dreyfus](https://github.com/dreyfus92), and [Jumper](https://github.com/jdtjenkins).
```

Would love to keep the content evergreen, meaning we don't want to refer to anything with time specific things, e.g. Astro's **latest** product